File: app/src/main/java/com/cyder/atsushi/youtubesync/json_data/Room.java
package com.cyder.atsushi.youtubesync.json_data;

import java.util.ArrayList;

/**
 * Created by atsushi on 2017/10/27.
 */

public class Room {
    public int id;
    public String name;
    public String description;
    public String key;
    public Video now_playing_video;
    public Video last_played_video;
    public ArrayList<User> online_users;
}

File: app/src/main/kotlin/com/cyder/atsushi/youtubesync/api/response/Room.kt
package com.cyder.atsushi.youtubesync.api.response

import com.google.gson.annotations.SerializedName

/**
 * Created by chigichan24 on 2018/01/18.
 */

data class Room(
        val id: Int,
        val name: String,
        val description: String,
        val key: String,
        @SerializedName("now_playing_video") val nowPlayingVideo: Video?,
        @SerializedName("last_played_video") val lastPlayedVideo: Video?,
        @SerializedName("online_users") val onlineUsers: List<User>?
)